pkg/cmd/translate.go

```diff
@@ -17,12 +17,10 @@ limitations under the License.
 package cmd
 
 import (
-	"fmt"
-	"log"
-	"os"
-
+	"github.com/deb-sig/double-entry-generator/pkg/cmd/validator"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"log"
```

Comment:
Please keep log in the first import group.

Reply:
`gofmt` ?

Reply:
Yep

Reply:
Fixed.